v0/destinations/blueshift/data/blueshiftIdentifyConfig.json

```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "destKey": "email",
+    "sourceKeys": ["traits.email", "context.traits.email"],
+    "required": true
+  },
+  {
+    "destKey": "customer_id",
+    "sourceKeys": [
+      "userId",
+      "traits.userId",
+      "traits.id",
+      "context.traits.userId",
+      "context.traits.id",
+      "anonymousId"
+    ],
+    "required": true
+  },
+  {
+    "destKey": "phone_number",
+    "sourceKeys": [
+      "traits.phoneNumber",
+      "traits.phone",
+      "context.traits.phoneNumber",
+      "context.traits.phone"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "firstname",
+    "sourceKeys": [
+      "traits.firstName",
+      "context.traits.firstName",
+      "traits.firstname",
+      "context.traits.firstname"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "lastname",
+    "sourceKeys": [
+      "traits.lastName",
+      "context.traits.lastName",
+      "traits.lastname",
+      "context.traits.lastname"
+    ],
+    "required": false
+  },
+  {
+    "destKey": "gender",
+    "sourceKeys": ["traits.gender", "context.traits.gender"],
+    "required": false
+  },
+  {
+    "destKey": "custom_attributes",
+    "sourceKeys": ["traits", "context.traits"],
+    "required": false
+  }
+]
```

Comment:
Prioritize context.traits items as it is an identifyConfig json.

Reply:
Done